netjsonconfig/__init__.py

```diff
@@ -15,5 +16,8 @@ def get_backends():
     }
 
     for entry_point in iter_entry_points('netjsonconfig.backends'):
-        default.update({entry_point.name.lower(): entry_point.load()})
+        try:
+            default.update({entry_point.name.lower(): entry_point.load()})
+        except ImportError as e:
+            print("Error loading backend {}".format(entry_point.name.lower()), file=sys.stderr)
```

Comment:
Shouldn't this rather use the python logging framework?

Reply:
[django-netjsonconfig](https://github.com/openwisp/django-netjsonconfig/blob/03a9ef8f282e6de57e939cc47733230cbfdcf1b8/django_netjsonconfig/utils.py#L9] uses

    import logging
    logger = logging.getLogger(__name__)
    logger.warning("Grrrrrr")

Reply:
Ok I'll change this to be a logger on the console :+1: